File: Software Development Module/Revision/Exam2015/SchoolSupplies.java
/*
SchoolSupplies.java
Billy Cussen
10/08/2020
*/

public class SchoolSupplies{

    //Data Members
    private int[] supplyQuantity;
    private double[] supplyCost;
    private String total;

    public SchoolSupplies(){
        this.total = "Total Cost of these Supplies is: ";
    }

    //Setter, Compute and Getter
    public String calculateSchoolSupplies(int[] supplyQuantity, double[]supplyCost){
        this.supplyQuantity = supplyQuantity;
        this.supplyCost = supplyCost;
        double numTotal = 0.0;

        for(int i = 0; i < this.supplyQuantity.length; i++){

            numTotal += this.supplyQuantity[i] * this.supplyCost[i];

        }
        this.total += numTotal;
        return this.total;
    }

}

File: Python/Software Development Module - Python/Revision/Exam2015/SchoolSupplies.py
"""
SchoolSupplies.py
Billy Cussen
28/10/2020
"""
class mySchoolSupplies:
    def __init__(self, supplyQuantity, supplyCost):
        self.supplyQuantity = supplyQuantity
        self.supplyCost = supplyCost

    def calculateSupplies(self):
        cost = 0.0
        iterator = 0
        for e in self.supplyQuantity:
            cost = cost + (self.supplyQuantity[iterator]*self.supplyCost[iterator])
            iterator=iterator+1

        return "Total is: "+str(cost)